CMakeLists.txt

```diff
@@ -126,8 +132,15 @@ set(NGRAPH_INSTALL_LIB "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
 set(NGRAPH_INSTALL_INCLUDE "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR}")
 set(NGRAPH_INSTALL_DOC "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DOCDIR}")
 set(NGRAPH_INSTALL_BIN "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}")
-set(CMAKE_INSTALL_RPATH "$ORIGIN")
+set(NGRAPH_INSTALL_BACKEND "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BACKEND_DIR}")
+set(NGRAPH_INSTALL_TEST "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_TEST_DIR}")
+if (DEFINED NGRAPH_RPATH)
+    set(CMAKE_INSTALL_RPATH "$ORIGIN:${NGRAPH_RPATH}")
```

Comment:
If the user is specifying an RPATH then should it take precedence over $ORIGIN? With this line it will search $ORIGIN first.